mapbox/50-states-project
# Store locations from geocoder input

Because we're using react now, storing locations now requires reading/writing from/to an API. This creates one.

To test this out:

- `git checkout store-locations` & `git pull`
- `cd server` (that's where the express server is located) & `npm start`
- In a new terminal window, open up the project at the base (ie `~/50-states-project`)
- `npm start`

Open up the app in your browser at localhost:8080. You should see some points already on the map. They are being read from the backend (you can check them out in geojson form at http://localhost:5000/places).

To add new points to the map, you can search for a place using the geocoder, and the result you click on will be added as a point. This data will persist since it's dynamically updating the geojson being served out as the backend.

@chrisrieth @lemwerks let me know if you have questions.

Closes #6
